x/programs/cmd/simulator/build.rs

```diff
@@ -36,7 +36,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let go_build_output = Command::new("go")
         .arg("build")
         .arg("-tags")
-        .arg("debug")
```

Comment:
don't we want this to be debug so that the host will log?

Reply:
Good catch! This was a debugging leftover, I probably thought to have edited the `simulator` command invokation.